src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -79,19 +78,19 @@ public StudyController(StudyService studyService) {
                                                   @RequestHeader("userId") String userId) {
         Mono<BasicStudyInfos> createStudy = studyService.createStudy(studyName, Mono.just(caseFile), description, userId, isPrivate)
                 .log(StudyService.ROOT_CATEGORY_REACTOR, Level.FINE);
-        return ResponseEntity.ok().body(studyService.assertStudyNotExists(studyName, userId).then(createStudy));
+        return ResponseEntity.ok().body(createStudy);
     }
 
-    @GetMapping(value = "/{userId}/studies/{studyName}")
```

Comment:
why still userId in the path ?

Reply:
it's used by assertUserAllowed function

Reply:
getCurrentUserStudy function needs that parameter